dialog/color_picker.go

```diff
@@ -192,7 +192,7 @@ func (p *colorAdvancedPicker) CreateRenderer() fyne.WidgetRenderer {
 	))
 
 	r := &colorPickerRenderer{
-		BaseRenderer:      internalwidget.NewBaseRenderer([]fyne.CanvasObject{contents}),
+		SimpleRenderer:    internalwidget.NewSimpleRenderer(contents),
```

Comment:
I think you could use the renderer from the public widget package directly here 🤔, so we don't need to import the internal widget package.

Reply:
Not exactly, there is no type exported following the previous review. But I can make it a generic WidgetRenderer